packages/preview/ugentthesis/0.1.3/typst.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "ugentthesis"
```

Comment:
Unless this package is officially endorsed by the University, I will ask you to change its name to something less canonical, to fit [our naming guidelines](https://github.com/typst/packages/blob/0488b38bb7fdfbce81981803ebbcf5612f8919f8/docs/manifest.md#naming-rules). If it is an official package, please let me know, we have a specific verification process to follow.